Decred Wallet/Extensions/UIStackView.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  UIStackView.swift
+//  Decred Wallet
+//
+// Copyright (c) 2018-2019 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+import UIKit
+
+extension UIStackView {
+    // Embed the UIstackiew components in a parent UIview with rounded edges as UIstackview does disregards it's layer.cornerRadius directive
+    func cornerRadius(_ radius: CGFloat) {
```

Comment:
function names should be verbs, rather than nouns: `setCornerRadius` rather than `cornerRadius`.

Reply:
resolved in 174e2f3